src/rules/selector-type-case/__tests__/index.js

```diff
@@ -0,0 +1,76 @@
+import {
+  ruleTester,
+} from "../../../testUtils"
+import rule, { ruleName, messages } from ".."
+
+const testRule = ruleTester(rule, ruleName)
+
+testRule("lower", tr => {
+    
+  tr.ok("a { display: inline-block; }")
+  tr.ok("div { display: inline-block; }")
+  tr.ok("body { display: inline-block; }")
+  tr.ok("li { display: inline-block; }")
+
+  tr.ok("a::before { content: \"some content\"; }")
+  tr.ok("div::before { display: inline-block; }")
+  tr.ok("body::before { display: inline-block; }")
+  tr.ok("li::before { display: inline-block; }")
+
+  tr.ok("&a { display: inline-block; }")
+  tr.ok("&li { display: inline-block; }")
+
+  tr.ok(".foo { display: inline-block; }")
+  tr.ok("#bar { display: inline-block; }")
+
+  tr.ok(".FOO { display: inline-block; }")
+  tr.ok("#BAR { display: inline-block; }")
+
```

Comment:
Can we get a tests for a compound selector, complex selector, single nesting selector and a selector list? e.g.

```js
tr.ok("a.FOO {}")
tr.ok("a b {}")
tr.ok("a { & b {}}")
tr.ok("a, b, * {}")
```